pandas/core/internals.py

```diff
@@ -1199,20 +1199,35 @@ def _interpolate_with_fill(self, method='pad', axis=0, inplace=False,
         blocks = [self.make_block_same_class(values, ndim=self.ndim)]
         return self._maybe_downcast(blocks, downcast)
 
+    # TODO: ignoring `values`?
     def _interpolate(self, method=None, index=None, values=None,
                      fill_value=None, axis=0, limit=None,
                      limit_direction='forward', limit_area=None,
                      inplace=False, downcast=None, mgr=None, **kwargs):
         """ interpolate using scipy wrappers """
 
         inplace = validate_bool_kwarg(inplace, 'inplace')
-        data = self.values if inplace else self.values.copy()
 
         # only deal with floats
-        if not self.is_float:
```

Comment:
i prefer to import these rather than use ct

Reply:
Darn, I was hoping that would catch on (I don't like the giant namespaces).  Will change.